packages/postgres-database/src/simcore_postgres_database/migration/versions/5860ac6ad178_adds_snapshots_table.py

```diff
@@ -0,0 +1,51 @@
+"""adds snapshots table
+
+Revision ID: 5860ac6ad178
+Revises: c2d3acc313e1
+Create Date: 2021-08-11 13:21:55.415592+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5860ac6ad178"
+down_revision = "c2d3acc313e1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "snapshots",
```

Comment:
Ok, so I think I might slowly be getting it. so it is not a parametrization of a project but really a project snapshot at a moment in time? is that right
Then I would still call the table something like ```project_snapshots```. for the sake of sorting, and also making things go together.

Reply:
I renamed as ``projects_snapshots``